modules/graceful/manager_windows.go

```diff
@@ -43,7 +43,7 @@ type Manager struct {
 	runningServerWaitGroup sync.WaitGroup
 	createServerWaitGroup  sync.WaitGroup
 	terminateWaitGroup     sync.WaitGroup
-	shutdownRequested	   chan struct{}
+	shutdownRequested      chan struct{}
```

Comment:
Please remove unrelated change.

Reply:
@guillep2k should I send a own PR for this one-line change?!?

Reply:
It's just a formation issue and since **this** PR is smal I think it fits in well

Reply:
> 
> 
> @guillep2k should I send a own PR for this one-line change?!?

@6543 I guess this change comes from `make fmt` itself so it should be OK. But if it was not, what I usually do is to skip unrelated changes from `git add`.